### PR TITLE
Created the foundation for the continued account creation page

### PR DESCRIPTION
**Pull request by carsonwolber**

### CODE REVIEW

**Comment by carsonwolber** [comment]
Screenshot 2024-06-28 at 4 18 00 PM

File: frontend/src/HomePage/SignHandlers/SignUp.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.topbar {
```
Comment (William-La) [inline frontend/src/HomePage/SignHandlers/SignUp.css:1]: nit: could we use camelCase for these class names? camelCase is pretty widely used and helps improve readability.

File: frontend/src/ProfilePage/FormPages/PageOne.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.form-page label {
```
Comment (William-La) [inline frontend/src/ProfilePage/FormPages/PageOne.css:1]: nit: see the camelCase comment from earlier. you could go with hyphens as well-- let's just be consistent

**Review by William-La — APPROVED** [review]
The create account page in your screenshot looks really good! Thanks for including a screenshot as a part of your test plan.

Looks like it's a lot more app set up in this PR. React files look pretty good to me!

File: frontend/src/HomePage/SignHandlers/SignUp.css (hunk 1 above)
Comment (pangers) [reply frontend/src/HomePage/SignHandlers/SignUp.css:1]: As mentioned in your other comment, I think using hyphens for CSS seems to be the common from my observations so I'd go with hyphens (but use camel case in other code)

**Pull request merged**